src/serac/numerics/functional/tuple.hpp

```diff
@@ -260,6 +260,46 @@ SERAC_HOST_DEVICE constexpr const auto& get(const tuple<T...>& values)
   }
 }
 
+/**
+ * @brief a function intended to be used for extracting the ith type from a tuple
+ * get<i>(my_tuple) returns a reference, so it requires additional postprocessing to
+ * extract the underlying type (without a reference)
```

Comment:
```suggestion
 * @brief a function intended to be used for extracting the ith type from a tuple.
 * As opposed to `get<i>(my_tuple)` which returns a reference, so it requires additional
 * postprocessing to extract the underlying type (without a reference).
```

If this is true then i feel this is more understandable.

Reply:
I'll take a stab at improving the wording. Without punctuation on the first line, it does look like a confusing run-on sentence, which was not intended.